docs/tutorials/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,535 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Advanced VQE Options\n",
+    "\n",
+    "In the first algorithms tutorial, you learned how to set up a basic [VQE](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VQE.html) algorithm. Now, you will see how to provide more advanced configuration parameters to explore the full range of capabilities of Qiskit's variational algorithms: [VQE](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VQE.html), [QAOA](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.QAOA.html) and [VQD](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.VQD.html) among others. In particular, this tutorial will cover how to set up a `callback` to monitor convergence and the use of custom `initial point`s and `gradient`s."
```

Comment:
>  full range of capabilities of Qiskit's variational algorithms:

Maybe we should change this we could do `full range of the variational algorithms provided` or implemented - just not  `Qiskit's` - we could do `qiskit_algorithms' variational algorithms` but since this is all about qiskit_algorithms it seemed less than needed to be so specfiic